spec/model/relation_spec.rb

```diff
@@ -135,6 +135,7 @@ class User < Foo::Model; end
           stub.get("/users?what=4&where=3") { ok! [{ id: 3, fullname: "Maeby Fünke" }] }
           stub.get("/users?what=2") { ok! [{ id: 2, fullname: "Lindsay Fünke" }] }
           stub.get("/users?where=6") { ok! [{ id: 4, fullname: "Tobias Fünke" }] }
+          stub.get('/bar/users') { ok! [] }
```

Comment:
`Bar::User.scoped#respond_to?` is set to do a `fetch` under the hood, which is why this stub was necessary. I gave the model its own `collection_path` to make sure this stub wouldn't interfere with the other model in this context.